molo/core/models.py

```diff
@@ -141,6 +141,15 @@ class SiteSettings(BaseSetting):
             "Global GA Tag Manager tracking code (e.g GTM-XXX) to be used"
             " to view analytics on more than one site globally")
     )
+    google_search_console = models.CharField(
+        verbose_name=_('Google Search Console'),
+        max_length=255,
+        null=True,
+        blank=True,
+        help_text=_(
+            "The Google Search Console tracking code (e.g GTM-XXX) to be used"
+            " to view search performance")
```

Comment:
Are you sure this Help text is correct? Perhaps I'm misunderstanding the ticket but I thought you needed to add a verification code not a tracking code? The one I added to the corporate site didn't start with `GTM-...`

Reply:
@KaitCrawford I updated the help text